packages/TelegramClientTests-UI.package/TCTUChatInfoPageTests.class/instance/openAnyChatThatSatisfies..st

```diff
@@ -0,0 +1,8 @@
+navigating
+openAnyChatThatSatisfies: aClosure
```

Comment:
I find `openAnyChatSatisfying: aClosure` sounds better

Reply:
I renamed it